pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -163,6 +169,8 @@ stdenv.mkDerivation rec {
     bzip2 fontconfig freetype gnutls libiconv lame libass libogg libssh libtheora
     libvdpau libvorbis lzma soxr x264 x265 xvidcore zlib libopus speex nv-codec-headers
   ] ++ optional openglSupport libGLU_combined
+    ++ optional libmfxSupport intel-media-sdk
+    ++ optional vpxSupport libaom
```

Comment:
Did you mean to use `libaomSupport` here?

Reply:
sorry just seeing this now since your comment happened after the merge. I think you may be right but I can't remember off hand, but it's highly likely to be a typo.